src/BookingStateWrapper/MultiCityTrip.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import * as React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+
+import BookingStateProvider from '../context/BookingState';
+import HasBooking_booking from './__generated__/HasBooking_booking.graphql';
+
+type Props = {
+  children: React.Node,
+  +booking: HasBooking_booking,
+};
+
+const MultiCityTrip = ({ children, booking }: Props) => (
+  <BookingStateProvider
+    hasBooking
+    isPastBooking={booking.isPastBooking}
+    isUrgent={false /*fixme*/}
```

Comment:
fixme?

Reply:
Technically this is fixed here: https://github.com/kiwicom/smart-faq/pull/654 I didn't copy that fix into this MR because it was not trivial and I wanted to make re-structuring the other PR as simple as possible. It's kinda ugly to have that here, but also, this component is not being used, so at least it won't cause issues

Reply:
ok, then please fix it soon

Reply:
:+1: I'll just have to cherry pick the commit that fixes it